changelog/unreleased/pull-2546

```diff
@@ -0,0 +1,8 @@
+Change: Exit status 2 when errors occured while reading source data. Previously, 
+		Restic just created the incomplete snapshot while exiting with status 0.
```

Comment:
"incomplete" is slightly ambiguous here: It could either mean that the snapshot creation skipped a few files or that the repository misses part of the snapshot data. The current wording is also unclear whether restic still creates a snapshot after this change or not.